packages/api/src/libs/synthPrices.ts

```diff
@@ -0,0 +1,52 @@
+import { createReferencePriceFeedForFinancialContract, Networker } from "@uma/financial-templates-lib";
+import winston from "winston";
+import assert from "assert";
+import type Web3 from "web3";
+
+type Config = {
+  cryptowatchApiKey?: string;
+  tradermadeApiKey?: string;
+  quandlApiKey?: string;
+  defipulseApiKey?: string;
+  priceFeedDecimals?: number;
+};
+type Dependencies = {
+  web3: Web3;
+};
+// cannot get type of Pricefeed, would be great if we could
+type PriceFeed = any;
+
+export default (config: Config = {}, { web3 }: Dependencies) => {
```

Comment:
nit: what's a dependency?  Is it another name for Lib? Or is it like an import that we've added state to? How does it differ from the stuff in appState?

Reply:
in my head i think of it like dependency injection, usually stateful classes, or things that could be mocked that are not just JSON. it might even be better to add the logger to the dependencies, not sure. I think i'll just simplify this to just web3.